mothership/holesky-devnet/values.yaml

```diff
@@ -21,3 +22,9 @@ blockscout:
 node:
   replicas: 2
   loadBalancerExternal: true
+
+opProposer:
+  replicas: 0
```

Comment:
This part overwrites https://github.com/planetarium/9c-infra/pull/1497/files#diff-2bf98a80a27a5317cf41b77d6c6ea635eb7a5f425acaaf5c92bde1fc87aa1ab8L43 this value right?